src/sources/community.js

```diff
@@ -190,6 +191,12 @@ class CommunityDataset extends Dataset {
     // name in the file basename.
     return [`auspice/${this.source.repoName}`, ...this.pathParts];
   }
+  lookupVersionInfo(versionDescriptor) {
```

Comment:
Community URLs already allow exact versions to be specified via a branch / commit SHA, e.g.:
```
/community/[narratives/]<orgName>/<repoName>@<branchName|SHA>[/optional/extra/bits]
```
I've set things up so that this doesn't interfere with parsing our own `@<versionDescriptor>` at the end of the path, but since we aren't collecting any community-based resources at the moment it's probably easiest to disable looking for a versionDescriptor (as I do with /fetch URLs). While I think we'll always support the current git-specific lookups there's no reason we can't also offer `@<versionDescriptor>` functionality for community datasets & narratives, especially as we'll have to implement similar logic to deal with core & staging narratives which are housed on GitHub.

Reply:
I've inverted my previous approach so that now version descriptors are only extracted for core datasets. This resolves a few other PR comments as well. Source/Resource classes can opt-in over time as desired.